engine.js

```diff
@@ -444,6 +445,41 @@ HMDAEngine.prototype.runSpecial = function(year) {
     });
 };
 
+/**
+ * Export errors in csv format for an individual edit
+ * @param {string} year         The specific year of the file specification to work with
+ * @param {string} errorType    The edit category. Valid values: 'syntactical', 'validity', 'quality', 'macro', 'special'
+ * @param {string} errorID      The ID of the edit to export
+ * @param {object} writeStream  Handle to a {@link https://nodejs.org/api/stream.html#stream_class_stream_writable_1|stream.Writable} instance to output to
+ * @see {@link CSVProcessor|CSVProcessor} for more info
+ */
+HMDAEngine.prototype.exportIndividual = function(year, errorType, errorID, writeStream) {
+    var csvProcessorIndividual = new CSVProcessor(year, writeStream, 'individual');
+    if (this.getErrors()[errorType][errorID]) {
+        var errorsIndividual = {};
+        errorsIndividual[errorID] = this.getErrors()[errorType][errorID];
+        csvProcessorIndividual.write(errorsIndividual);
+    }
+
+    csvProcessorIndividual.end();
+};
+
+/**
+ * Export errors in csv format for all errors of a specific type
+ * @param {string} year         The specific year of the file specification to work with
+ * @param {string} errorType    The edit category. Valid values: 'syntactical', 'validity', 'quality',
+ * @param {object} writeStream  Handle to a {@link https://nodejs.org/api/stream.html#stream_class_stream_writable_1|stream.Writable} instance to output to
+ * @see {@link CSVProcessor|CSVProcessor} for more info
+ */
+HMDAEngine.prototype.exportAll = function(year, errorType, writeStream) {
```

Comment:
I know we called it "export all" in our discussions, but since this function is limited to the errorType, should we change the name to `exportType`?